.github/scripts/codespell.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+SCAN_RESULT=`codespell -H --quiet-level=3 --skip="*.h,*.cmake,*.c,*.hpp,*.cpp,*.sh,*.py,test/supp/*.supp,./.venv" --ignore-words-list="ASSER,Tne,ba,BA"`
```

Comment:
is there like a lot issues in these files? why skipping so much?

for sure we want to skip `*.supp,./.venv`, the others, I'm not sure

you can shorten the list of skipped files and not fix any issues. The CI will fail then and we could observe the output of this script and make the decision what to do with issues (if any)